onedal/backend/data/data.cpp

```diff
@@ -160,6 +296,17 @@ PyObject *convert_to_numpy(const dal::table &input) {
             throw std::invalid_argument("oneDAL have don't table row major format");

```

Comment:
Strange statement. As I know oneDAL homogen table is support both row and column layout. If we decided not to support column layout in python side, my suggestion is to change wording in the error, because I can only guess the idea

Reply:
Applied